rendercv/data/models/rendercv_settings.py

```diff
@@ -0,0 +1,134 @@
+"""
+The `rendercv.models.rendercv_settings` module contains the data model of the
+`rendercv_settings` field of the input file.
+"""
+
+from typing import Optional
+
+import pydantic
+
+
+class RenderCVSettings(pydantic.BaseModel):
```

Comment:
Please update `RenderCVSettings` so that the YAML file looks like this:

```yaml
rendercv_settings:
  render_settings:
    ... # all available render command CLI options
```

Reply:
This might be a good Idea. Later we can add `filter` or `version` to reflect #164

Reply:
This is done. 

The render cli settings can be found in the following format:

```yaml
rendercv_settings:
  render:
    output_folder_name:
    # ... other options
```